ironic-images/ipa-debian-bookworm/ipa-debian-bookworm.yaml

```diff
@@ -0,0 +1,8 @@
+- imagename: ipa-debian-bookworm
+  elements:
+  - ironic-python-agent-ramdisk
+  - debian
+  - devuser
```

Comment:
```suggestion
```

Let's drop `devuser` for `dynamic-login`

Reply:
Switched